### PR TITLE
Added new companies

### PR DESCRIPTION
**Pull request by nyanag**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,10 +82,18 @@ Short description.
 
   Collaborated with NASA to develop a shower system that uses 90% less water. The shower has a purification system that makes it possible to reuse the same batch of water. The company now wants to make this technology - initially developed for space - available to consumers.
 
+- [Seabin Project](https://seabinproject.com/) | Australia | -
```
Comment (nglgzz) [inline README.md]: I searched and they started as an [Indiegogo campaign](https://www.indiegogo.com/projects/cleaning-the-oceans-one-marina-at-a-time#/) in 2016, so I think we can use this year as funding year.

File: README.md [hunk 2]
```diff
@@ -82,10 +82,18 @@ Short description.
...
+
+  To provide practical and tangible solutions to reduce the plastics in our oceans which is one of the world’s greatest problems.
```
Comment (nglgzz) [inline README.md]: I think the description could be improved to tell better what they're doing, and how far they're into the development of their solution. What about this instead:

```
Their mission is to reduce plastic in the oceans through a floating rubbish bin that 
collect pollutants on the water surface. As of October 2019 they have deployed 
719 Seabins and are capturing almost 3 tonnes of rubbish per day.
```

File: README.md [hunk 3]
```diff
@@ -124,6 +132,10 @@ Short description.
 
   Specialized in electric car manufacturing and through, its SolarCity subsidiary, solar panel manufacturing.
 
+- [Tessol](https://www.tessol.in/) | Mumbai, India | 2013
```
Comment (nglgzz) [inline README.md]: Not sure if this company classifies as clean-tech. 

From their website I cannot find anything related to sustainability. Their main goal seems to be `Redefining Cold Chain Logistics`, it might be that they end up making something with a good impact, which is great, but the main goal of the company doesn't seem to be related to improving the environment.

**Review by nglgzz — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (nyanag) [reply README.md]: Okay, I will add that. Thanks!

File: README.md (hunk 2 above)
Comment (nyanag) [reply README.md]: Agreed. Will make the changes

File: README.md (hunk 3 above)
Comment (nyanag) [reply README.md]: Yes okay, I've researched and added another company instead. I will send a PR.

Thanks for reviewing my PR
